xonsh/environ.py

```diff
@@ -687,10 +687,14 @@ def _is_executable_file(path):
 
 
 def yield_executables_windows(directory, name):
+    casefold_name = name.casefold()
     extensions = builtins.__xonsh_env__.get('PATHEXT')
     for a_file in os.listdir(directory):
         base_name, ext = os.path.splitext(a_file)
-        if name == base_name and ext.upper() in extensions:
+
+        if (
+            casefold_name == base_name.casefold() or casefold_name == a_file.casefold()
+        ) and ext.upper() in extensions:
```

Comment:
Yes. I hadn't used it before, but the python docs told me what it did. I was wondering if you had any specific reasons to use it here. I just tested this on my machine: "ß.txt" and "ss.txt" are two distinct files. Is there something I have misunderstood?